pkg/apis/pipeline/v1beta1/provenance.go

```diff
@@ -13,22 +13,20 @@ limitations under the License.
 
 package v1beta1
 
-// Provenance contains some key authenticated metadata about how a software artifact was
-// built (what sources, what inputs/outputs, etc.). For now, it only contains the subfield
-// `ConfigSource` that identifies the source where a build config file came from.
-// In future, it can be expanded as needed to include more metadata about the build.
-// This field aims to be used to carry minimum amount of the authenticated metadata in *Run status
-// so that Tekton Chains can pick it up and record in the provenance it generates.
+// Provenance contains metadata about resources used in the TaskRun/PipelineRun
+// such as the source from where a remote build definition was fetched.
+// This field aims to carry minimum amoumt of metadata in *Run status so that
+// Tekton Chains can capture them in the provenance.
 type Provenance struct {
-	// ConfigSource identifies the source where a resource came from.
-	ConfigSource *ConfigSource `json:"configSource,omitempty"`
+	// Source field identifies the source where a remote built script came from.
+	Source *Source `json:"source,omitempty"`
 }
 
-// ConfigSource identifies the source where a resource came from.
-// This can include Git repositories, Task Bundles, file checksums, or other information
-// that allows users to identify where the resource came from and what version was used.
-type ConfigSource struct {
-	// URI indicates the identity of the source of the config.
+// Source contains the information that can uniquely identify where a remote
+// built definition came from i.e. Git repositories, Tekton Bundles in OCI registry
+// and hub.
+type Source struct {
```

Comment:
How about using type alias to keep backward compatible? 
like
https://github.com/tektoncd/pipeline/blob/a127323da31bcb933a04a6a1b5dbb6e0411e3dc1/pkg/apis/pipeline/v1beta1/param_types.go#L141

Reply:
I think it's ok to do since this is a newly released alpha feature. We should definitely call it out in the release notes as a breaking change though

Reply:
Is Chains using this yet? We should also reach out to the Chains WG to make sure they are aware!